samples/04_gis_analysts_data_scientists/model_explainability_using_shap_for_tabular_data.ipynb

```diff
@@ -0,0 +1,534 @@
+{
```

Comment:
<p> method in <code>arcgis.learn</code> <strong>for </strong>MLModel and Fully.. (missing for)</p><p>The paramter explain -&gt; typo in parameter. Better to quote 'explain' in this line as well. same in next line / bullet.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/977/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>Done</p>

Reply:
<p>Done</p>